system/views/view.py

```diff
@@ -0,0 +1,84 @@
+import tkinter as tk
+from tkinter import ttk
+
+
+class View(tk.Tk):
+
+    def __init__(self, *args, **kwargs):
+        tk.Tk.__init__(self,  *args, **kwargs)
+
+        # creating a container
```

Comment:
Some of these comments may not be necessary, especially here where it's pretty obvious a container is being created.